activities/activity_laginhas.py

```diff
@@ -0,0 +1,48 @@
+#!/usr/bin/env python3
+
+"""
+Comparing differences for resampling methods: average & median.
+"""
+import grass.script as gs
+
+
+def run_resamplediffs(scanned_elev, env, **kwargs):
+    gs.run_command("g.region", res=12, flags="a", env=env)
```

Comment:
You need to use the code snippet I posted earlier instead of running g.region directly. This would influence the global region setting, which we are trying to avoid. I understand this is confusing, it's just handling region in Tangible Landscape is more complex.